index.js

```diff
@@ -499,20 +499,16 @@ function signAsync (opts) {
     })
     .then(function () {
       // Pre-sign operations
-      var promise = Promise.resolve()
-      if (opts.version ? compareVersion(opts.version, '1.1.1') >= 0 : true) {
+      if (opts.entitlements != null && (opts.version == null || compareVersion(opts.version, '1.1.1') >= 0)) {
```

Comment:
Thanks @develar 😸
Honestly the `null`/`undefined` in JavaScript still messes around my work.